chain/cmd/bandoracled2/run.go

```diff
@@ -82,6 +82,14 @@ func runCmd(c *Context) *cobra.Command {
 				return err
 			}
 			l := NewLogger(allowLevel)
+			executorName, executorURL, err := ParseExecutor(cfg.Executor)
+			if err != nil {
+				return err
+			}
+			c.executor, err = NewExecutor(executorName, executorURL)
```

Comment:
This assumes that all executor strings always have two parts: the name and the URL, which is likely not always the case. I think we can just have `NewExecutor` function just takes `cfg.Executor` directly.